traveler-ios-ui/TravelerKitUI/ViewControllers/BookingItemSearch/BookingItemQueryCategoryViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  BookingItemQueryCategoryViewController.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-11-15.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+public protocol BookingItemQueryCategoryViewControllerDelegate: class {
+    func bookingItemQueryCategoryViewController(_ controller: BookingItemQueryCategoryViewController, didFinishWith categories: [BookingItemCategory])
+}
+
+open class BookingItemQueryCategoryViewController: UITableViewController {
+    private let categories = BookingItemCategory.allCases
+    
+    public weak var delegate: BookingItemQueryCategoryViewControllerDelegate?
+    
+    public var selectedCategories: [BookingItemCategory] = []
+    
+    public override func viewDidLoad() {
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: CellIdentifiers.generic)
```

Comment:
@OmarPadierna Do we have to register table view cells manually or set the reuse identifier in the storyboard?

Reply:
@benRmc In my perspective, doing by code is more maintainable and easier to account for, especially if using a generic reuse identifier.  When using it from a contants file (or even class definition), you're sure to have the and use the same re-use identifier as expected.  When defining it in a storyboard, you lose that integrity as you won't know until runtime that there's a mismatch.